RUMM-558 Add Realm use case into the sample app (#378)
DataDog/dd-sdk-android: docs/rum_collection.md

                .build()
```

### SQLite

> Following SQLiteOpenHelper's [Generated API documentation][8], you only have to provide our implementation of the

Reviewer: ```suggestion
Following SQLiteOpenHelper's [Generated API documentation][8], you only have to provide the implementation of the
```

DatabaseErrorHandler -> `DatadogDatabaseErrorHandler` in the constructor.

> Doing so we will be able to detect whenever a database corruption was detected and send a relevant

Reviewer: ```suggestion
Doing this detects whenever a database is corrupted and sends a relevant
```
